src-tauri/src/node.rs

```diff
@@ -148,63 +173,75 @@ fn create_configuration<CS: ChainSpec + 'static>(
     let telemetry_endpoints = chain_spec.telemetry_endpoints().clone();
 
     // Default value are used for many of parameters
-    Ok(Configuration {
-        impl_name,
-        impl_version,
-        tokio_handle,
-        transaction_pool: Default::default(),
-        network,
-        keystore_remote,
-        keystore,
-        database: database_config(&config_dir, database_cache_size, &role),
-        state_cache_size: 67_108_864,
-        state_cache_child_ratio: None,
-        // TODO: Change to constrained eventually (need DSN for this)
-        state_pruning: PruningMode::ArchiveAll,
-        keep_blocks: KeepBlocks::All,
-        wasm_method: WasmExecutionMethod::Compiled,
-        wasm_runtime_overrides: None,
-        execution_strategies: ExecutionStrategies {
-            syncing: ExecutionStrategy::AlwaysWasm,
-            importing: ExecutionStrategy::AlwaysWasm,
-            block_construction: ExecutionStrategy::AlwaysWasm,
-            offchain_worker: ExecutionStrategy::AlwaysWasm,
-            other: ExecutionStrategy::AlwaysWasm,
+    Ok(SubspaceConfiguration {
+        base: Configuration {
+            impl_name,
+            impl_version,
+            tokio_handle,
+            transaction_pool: Default::default(),
+            network,
+            keystore_remote,
+            keystore,
+            database: database_config(&config_dir, database_cache_size, &role),
+            state_cache_size: 67_108_864,
+            state_cache_child_ratio: None,
+            // TODO: Change to constrained eventually (need DSN for this)
+            state_pruning: PruningMode::keep_blocks(1024),
+            keep_blocks: KeepBlocks::Some(1024),
+            wasm_method: WasmExecutionMethod::Interpreted,
```

Comment:
Ouch, no interpreted WASM!!!

Reply:
Since we removed `wasm exeuction compiled` from readme
https://github.com/subspace/subspace/pull/405/files#diff-cb44c4563e0dbee8cb7ed4e05a25e2708bd167ef5b59cabc8e9fcad2dbb70e73L39

I thought it would use the default.

and the default for `WasmExecutionMethod` was ->
```rust
impl Default for WasmExecutionMethod {
	fn default() -> WasmExecutionMethod {
		WasmExecutionMethod::Interpreted
	}
}
```

So we are using `compiled` and it is declared in somewhere else?